Src/Support/Google.Apis.Auth/OAuth2/TokenRefreshManager.cs

```diff
@@ -92,6 +92,16 @@ internal async Task<string> GetAccessTokenForRequestAsync(CancellationToken canc
                     // otherwise _refreshTask is updated in an incorrect order.
                     // And Task.Run also means it can be run here in the lock.
                     _refreshTask = Task.Run(RefreshTokenAsync);
+
+                    // Let's make sure that exceptions in _refreshTask are always observed.
+                    // Note that we don't keep a reference to this new task as we don't really
+                    // care about the errors. We just schedule it to run and that's enough for
+                    // avoiding exception observavility issues.
+                    Task.Run(() => _refreshTask.ContinueByMaybeHandlingExceptions(ex =>
+                    {
+                        _logger.Debug($"An error occured on a background token refresh task.{Environment.NewLine}{ex}");
```

Comment:
Is `_logger` always non-null?

An alternative to `Task.Run` would be to just call `_refreshTask.ContinueWith` (and a suitable TaskContinuationOptions). I think that would be more efficient, and more obvious in intent.